### ISSUE TITLE
Clarification for "InitAbilityActorInfo" vs "RefreshAbilityActorInfo"

### ISSUE THREAD

**Issue by H1X4Dev** [tranek/GASDocumentation#25]
From my initial point of view, this isn't really a standard way to create an ability system, yes it does display that you can move components around, however because that this is on Player State, rather than being in a Character actor (Monsters, NPC's, etc can use Ability System, it's not really limited to the player) it could yield undesired results if implemented differently

While I was trying to implement the GAS system into my project, I found that the PlayerController was always set to null, since I was using the component to be per-character, rather than per-player-state (Reference where it's used https://github.com/EpicGames/UnrealEngine/blob/release/Engine/Plugins/Runtime/GameplayAbilities/Source/GameplayAbilities/Private/AbilitySystemComponent_Abilities.cpp#L1164) thus any ability that tried to run was always being denied with "Can't activate LocalOnly or LocalPredicted ability %s when not local! Net Execution Policy is %d.".

This issue can be resolved by calling RefreshAbilityActorInfo() on PossessedBy() override your player actor.

TLDR: InitAbilityActorInfo is called automatically whenever the component is constructed, but if you have a player controller, it will not take it into the account, you must call RefreshAbilityActorInfo().

I haven't seen this being noted anywhere in the documentation, so hope the knowledge here would help someone that was stuck in the same boat.

Reference commit: https://github.com/tranek/GASDocumentation/commit/831ec2e66b59db75f1205095819ed6fbdf81f0f4

**Comment by tranek** [comment]
Adding information about where to call `InitAbilityActorInfo()` when the ASC is on the Character for multiplayer games is a good suggestion. A lot of people ask about that "Can't activate LocalOnly..." error. This would be good to link to.

In my projects where I have the ASC on the player-controlled Character class, I call `InitAbilityActorInfo()` in both `PossessedBy()` (server-only) and in `APlayerController::AcknowledgePossession()` (client). Simply calling `InitAbilityActorInfo()` in `PossessedBy()` alone is not enough to fix that error. They both call `AbilityActorInfo->InitFromActor()` but `InitAbilityActorInfo()` does additional code after that.

Since I don't have an example from Epic of the "Epic way" of doing it and this method works for me, this is what I will include in the docs. It will be included in the 4.25 update of the docs.
